src/style.css

```
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+.wrapper {
+  width: 100%;
+}
+
+.search-nav {
+  width: 100%;
+  position: relative;
+}
+
+.search-nav .search-bar {
```

```suggestion
.search-bar {
```
There's no need to use this combination. You can only give styles to needed class